src/DotNetBoilerplate.Api/Organizations/GetOrganizationEndpoint.cs

```diff
@@ -1,12 +1,13 @@
 ﻿using DotNetBoilerplate.Application.Organizations.Read;
 using DotNetBoilerplate.Shared.Abstractions.Commands;
+using DotNetBoilerplate.Shared.Abstractions.Queries;
 using Microsoft.AspNetCore.Builder;
 using Microsoft.AspNetCore.Http.HttpResults;
 using Microsoft.AspNetCore.Mvc;
 
 namespace DotNetBoilerplate.Api.Organizations
 {
-    public class ReadOrganizationEndpoint : IEndpoint
+    public class GetOrganizationEndpoint : IEndpoint
```

Comment:
GetOrganizationS (uważac na liczbę mnogą) albo żeby się nie mylić polecam nazywanie tam gdzie są listy z przedrostkiem browse np. BrowseOrganizations - wtedy się ciężko pomylić. To samo tyczy się nazw query i handlerów więc nie będę już tego zaznaczał potem